tripy/tripy/flat_ir/ops/constant.py

```diff
@@ -46,34 +45,35 @@ def to_mlir(self, operands):
         from tripy.backend.mlir import utils as mlir_utils
 
         # TODO(#189): Remove explicit copy to host for constants
-        if isinstance(self.data, Array):
-            assert self.data.dtype == self.outputs[0].dtype
-            memref_value = self.data.memref_value
-            if self.data.device.kind == "gpu":
-                memref_value = mlir_utils.MLIRRuntimeClient().copy_to_host(
+        if isinstance(self.data, runtime.MemRefValue):
+            runtime_client = mlir_utils.MLIRRuntimeClient()
+            memref_value = self.data
```

Comment:
Let's use `memref` instead of `memref_value` to be consistent as unlike MLIR-TensorRT, Tripy need not make such a distinction.

Reply:
Fixed, thanks